README.md

```diff
@@ -1,15 +1,54 @@
-# [![Build Status](https://travis-ci.com/SUSE/catapult.svg?branch=master)](https://travis-ci.com/SUSE/catapult) catapult: [SCF](https://github.com/SUSE/scf) on top of [Kind](https://github.com/kubernetes-sigs/kind) (and others..)
+# [![Build  Status](https://travis-ci.com/SUSE/catapult.svg?branch=master)](https://travis-ci.com/SUSE/catapult) CI implementation for [KubeCF](https://github.com/SUSE/kubecf), [SCF](https://github.com/SUSE/scf) & [Stratos](https://github.com/cloudfoundry/stratos)
 
-Get your local development [SCF](https://github.com/SUSE/scf) install with:
 
-    $> docker run -v /var/run/docker.sock:/var/run/docker.sock -ti --rm splatform/catapult:latest dind
-    
-Check out [Run in Docker](https://github.com/SUSE/catapult/wiki/Run-in-Docker) page on the wiki for more options.
- 
-## Description
+    $> git clone https://github.com/SUSE/catapult.git && cd catapult
+    $> make all
+
+This will start a local [Kind](https://github.com/kubernetes-sigs/kind) cluster
+and deploy scf on top of it. To remove everything, run `make clean`.
+
+Check the [First steps wiki page](https://github.com/SUSE/catapult/wiki/First-steps).
+
+
+# Description
+
+Catapult is a CI implementation that is from the ground-up designed to work
+locally. This allows iterating and using it for manual tests and development of
+the product, in addition to running it in your favourite CI scheduler
+(Concourse, Gitlab…).
+
+Catapult supports several k8s backends: can create CaaSP4, GKE, EKS clusters on its
+own, and you can bring your own cluster with the "imported" backend.
```

Comment:
I would add a reference in the wiki about this feature

Reply:
Instead of in the readme, or in addition to the readme?

Reply:
Updated https://github.com/SUSE/catapult/wiki/Imported